### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by plicease**
including details about using ALIEN_INSTALL_TYPE as an opt-in (#104)

### CODE REVIEW

**Comment by mohawk2** [comment]
Documentation commit (last one) looks good. The PR is now showing as having conflicts.

File: lib/Alien/Build/Manual/Alien.pod [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+=over 4
+
+=item The Alien Installer (configure / build-time)
+
+L<Alien::Build> and L<alienfile> are used to detect and install aliens. They are further documented in
+L<Alien::Build::Manual::AlienAuthor>.
+
+=item The Alien Runtime (runtime)
+
+L<ALien::Base> is the base class for aliens in the C<Alien-Build> system. Its use by Alien consumers
```
Comment (Grinnz) [inline lib/Alien/Build/Manual/Alien.pod]: typo: `L<Alien::Base>`

File: lib/Alien/Build/Plugin/Prefer/BadVersion.pm [hunk 2]
```diff
@@ -15,8 +15,9 @@ use Carp ();
 
 =head1 DESCRIPTION
 
-This plugin allows you to easily filter out known bad versions of libraries.  You need a Prefer plugin
-that filters and sorts files first.  You may specify the filter in one of three ways:
+This plugin allows you to easily filter out known bad versions of libraries in a share install.
+It doesn't effect a system install at all.  You need a Prefer plugin that filters and sorts files
```
Comment (Grinnz) [inline lib/Alien/Build/Plugin/Prefer/BadVersion.pm]: affect

File: lib/Alien/Build/Plugin/Prefer/GoodVersion.pm [hunk 3]
```diff
@@ -15,9 +15,10 @@ use Carp ();
 
 =head1 DESCRIPTION
 
-This plugin allows you to specify one ore more good versions of a library.  This plugin does
-the opposite of the C<Prefer::BadVersion> plugin.  You need need a Prefer plugin that filters
-and sorts files first.  You may specify the filter in one of three ways:
+This plugin allows you to specify one ore more good versions of a library.  This doesn't effect
```
Comment (Grinnz) [inline lib/Alien/Build/Plugin/Prefer/GoodVersion.pm]: affect

**Pull request merged**